### PR TITLE
[misc] tidy imports

### PR DESCRIPTION
**Pull request by ajm188**
## Description

Removes duplicate imports and tidies imports in any file I touched along the way.

## Related Issue(s)

#13877 

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
Thanks for doing this! LGTM

File: go/test/endtoend/messaging/message_test.go [hunk 1]
```diff
@@ -30,14 +30,15 @@ import (
 
 	"vitess.io/vitess/go/mysql"
 	"vitess.io/vitess/go/sqltypes"
+	"vitess.io/vitess/go/vt/log"
+	"vitess.io/vitess/go/vt/vtgate/vtgateconn"
+
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+
```
Comment (dbussink) [inline go/test/endtoend/messaging/message_test.go:38]: Seems strange to break these up into separate blocks if they are all for internal vitess imports?

Comment (ajm188) [reply go/test/endtoend/messaging/message_test.go:38]: in most other files we tend to separate imports from the generated protobufs into their own blocks

Comment (dbussink) [reply go/test/endtoend/messaging/message_test.go:38]: We seem very inconsistent about it though? Especially here where the protobuf ones split up multiple vitess import blocks and isn't entirely separate on their own? Anyway, just seems like churn for something we're don't seem to have standardized. And if we should, can we trick something like `goimports` into enforcing a consistent style? (I don't really even care that much about the style really).

**Review by dbussink — APPROVED** [review]

**Pull request merged**